cu-calendar/cu-calendar-test.tex

```diff
@@ -0,0 +1,28 @@
+\documentclass[12pt]{article}
+\usepackage{cu-calendar}
+\begin{document}
+
+\makeatletter
+\def\assertSame#1#2{%
+    \edef\@aa{#1}%
+    \edef\@bb{#2}%
+    \ifx\@aa\@bb\else
+        \errmessage{TEST FAILED: #1: "\@aa" expected "\@bb"}%
+    \fi
+}%
+
+\assertSame{\cu@jdnFromJulianDate{2016}{3}{30}}{2457491}
+\assertSame{\cu@jdnFromGregorianDate{2016}{3}{30}}{2457478}
+
+\assertSame{\cu@calendarJulian{2457491}}{2016-3-30}
+\assertSame{\cu@calendarGregorian{2457478}}{2016-3-30}
+
+\assertSame{\cu@julianFromGregorian{2016}{3}{30}}{2016-3-17}
+\assertSame{\cu@gregorianFromJulian{2016}{3}{17}}{2016-3-30}
+
+\assertSame{\cuJulianDate{2016}{3}{30}}{17 ма́рта л. 2016}
+\assertSame{\cuGregorianDate{2016}{3}{30}}{30 ма́рта л. 2016}
+
```

Comment:
more tests added, see https://github.com/slavonic/cu-tex/blob/flatten-file-tree/tests/cu-calendar-test.tex#L46

note that "cuDate" just formats data, so \cuDate{2016-04-33} is perfectly acceptable and will be formatted
as April 33rd. Same with \cuDate{1582-10-14}. What happens during the conversion julian <->gregorian is
a different story. This implementation assumes that dates before split are identical.

Date conversion works in a "soft" manner, meaning that invalid input will be normalized during the conversion
(32 Jan will work as if it is 1 Feb). This needs to be documented.